gatsby-node.js

```diff
@@ -33,9 +34,17 @@ exports.createPages = ({ graphql, actions }) => {
       const postEdges = result.data.allContentfulPage.edges
 
       postEdges.forEach(edge => {
+        var component;
+        if(edge.node.slug.includes("careers")){
```

Comment:
While I'd be okay on using this implementation as a first pass through to production, I wouldn't consider the work complete without moving this into the component switch statement, and creating a content type in Contentful.

Going down the switch route would mean you don't introduce a new careers template, don't rely on URLs which can be changed in the admin by users who may not be aware of the magic here, and enable you to include careers listings on other pages.

Reply:
:+1:  for removing the magic insert based on path